doc/linters.md

```diff
@@ -292,6 +292,50 @@ Depending on undefined task: compile
 Cyclic task dependency: a -> b -> a
 ```
 
+### Bb.edn global requires
+
+*Keyword:* `:bb.edn-global-requires`
```

Comment:
I think this could be a more general linter with a more general name. Instead of having a linter for each un-expected top level key, I think we could have `:bb.edn-unexpected-key` to lint for `:init`, `:requires`, and whatever more.

Reply:
Makes sense. What's your opinon on keeping the error message specific to the key in question? (eg being explicit about `:requires` belonging within the task map). i guess the mistake is quite clear for most cases, otoh i can't recall ever complaining about an error message or warning being too specific...

Reply:
Yeah, specific error message is helpful, agree